client/my-sites/checkout/composite-checkout/test/composite-checkout.js

```diff
@@ -534,15 +534,13 @@ describe( 'CompositeCheckout', () => {
 			fireEvent.click( await screen.findByText( 'Continue' ) );
 
 			// Wait for the validation to complete
-			await waitForElementToBeRemoved( () => screen.queryByText( 'Updating cart…' ) );
+			await waitForElementToBeRemoved( () => screen.queryAllByText( 'Please wait…' ) );
```

Comment:
This change is needed because I reversed the order of operations so that now we validate the tax info _before_ we update the cart.